hr_salary_rule_reference/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2013 Savoir-faire Linux
```

Comment:
Should be © 2018 <your name>.

Reply:
I did not do any customizations on this file, so i think it's good to keep it like that